lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -33,10 +33,7 @@ import 'package:redux/redux.dart';
 part 'over_react_redux.over_react.g.dart';
 
 @PropsMixin(keyNamespace: '')
-abstract class _$ConnectPropsMixin implements UiProps {
-  @override
-  Map get props;
-
+mixin ConnectPropsMixin on UiProps {
```

Comment:
@greglittlefield-wf this change is breaking `over_react_redux` tests.  If I update the test fixture to the new boilerplate, the tests pass. 

However, IIRC - this is unexpected since props mixins should be backwards compatible, correct?